### PR TITLE
Atom plugin !

### PR DESCRIPTION
**Pull request by vindarel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ sellers who aren't evil for physical resources.
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
 
-- [Awesome Common Lisp](https://github.com/sindresorhus/awesome)
```
Comment (rudolfochrist) [inline README.md]: Why is this removed?

**Comment by vindarel** [comment]
hello @CodyReichert , shall we merge this ?

**Comment by vindarel** [comment]
hello @CodyReichert, now that you'r back, what about this one ?
